src/coreclr/jit/compiler.h

```diff
@@ -658,6 +658,8 @@ class LclVarDsc
 private:
     unsigned char lvIsNeverNegative : 1; // The local is known to be never negative
 
+    unsigned char lvIsSpan : 1; // The local is a Span<T>
```

Comment:
would be nice to check that a new bit field doesn't mess the overall type layout?

Reply:
The plan is to remove lvIsNeverNegative (maybe in this PR too) so it shouldn't affect the layout.

Reply:
ah, makes sense then